www/layouts/contact/section.html

```diff
@@ -0,0 +1,90 @@
+
+<div id="contact-main-section" class="page-single">
+  <div class="px-lg-5 mt-6">
+    <div class="row">
+      <div class="col-12">
+        <h1>Contact Us</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-lg-6">
+        <div class="pr-lg-6 pr-4">
+          <div class="faq-section">
+            <h3>Frequently asked questions</h3>
+            <div class="faq">
+              <p class="question">
+                <b>What is MIT OpenCourseWare?</b>
+              </p>
+              <p class="answer">
+                MIT OpenCourseWare is a free and open publication of material from thousands of MIT courses,
+                covering the entire MIT curriculum and used by millions of learners and educators around the
+                world.
+              </p>
+            </div>
+            <div class="faq">
+              <p class="question">
+                <b>Do I have to register or sign up?</b>
+              </p>
+              <p class="answer">
+                No enrollment or registration. Freely browse and use OCW materials at your own pace. There's
+                no signup, and no start or end dates.
+              </p>
+            </div>
+            <div class="faq">
+              <p class="question">
+                <b>Why is OCW made free to the public?</b>
+              </p>
+              <p class="answer">
+                Knowledge is your reward. Use OCW to guide your own life-long learning, or to teach others. We
+                don't offer credit or certification for using OCW.
+              </p>
+            </div>
+            <div class="faq">
+              <p class="question">
+                <b>Are there any restrictions on use of OCW materials?</b>
+              </p>
+              <p class="answer">
+                Made for sharing. Download files for later. Send to friends and colleagues. Modify, remix, and
+                reuse (just remember to cite OCW as the source.)
+              </p>
+            </div>
+            <div class="faq">
+              <h3>Still have questions?</h3>
+              <p>
+                Please see our
+                <a href="https://ocw.mit.edu/help/">Help pages for answers to many common questions</a>
+                about OCW.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-6">
+        <div>
+          <img
+            width="100%"
+            src="/images/contact/frequently-asked-questions.jpg"
+            alt="Discover OER on MIT OpenCourseWare"
+          />
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-12">
+        <h3>Get in touch</h3>
+        <p>
+          If you did not find the answer in the <a href="https://ocw.mit.edu/help/">OCW help pages</a>, please fill out the form below
+          and a staff member we will respond as soon as possible.
+        </p>
+      </div>
```

Comment:
Every thing looks good to me. Just one small thing in the invision design the `Get in touch` block and `Still have questions` block probably were merged and the current design does not reflect that.
Also the contact form does not have a heading in the design.


<img width="1205" alt="Screenshot 2022-01-21 at 3 48 49 PM" src="https://user-images.githubusercontent.com/87968618/150514925-bf9e2781-0979-43f2-85d6-b168be91e4e2.png">

Reply:
Yeah, just saw it in the design. I'll change it.

Yeah, the contact form does not have a heading but Wufoo form does not allow us to create a form without heading, not sure how it is done in legacy site.